src/client/js/Panels/PropertyEditor/PropertyEditorPanelController.js

```diff
@@ -66,19 +66,10 @@ define(['js/logger',
             self._logger.debug('activeSelection changed', activeSelection);
             var activeNodeId = WebGMEGlobal.State.getActiveObject();
             if (activeSelection && activeSelection.length > 0) {
-                if (activeSelection.length === 1) {
-                    //https://github.com/webgme/webgme/issues/326
-                    if (self.startsWith(activeSelection[0], activeNodeId)) {
```

Comment:
Is this issue not valid anymore? Oh I see you removed the pushs.. Did you confirm that it worked as expected though?